core/src/main/java/tech/ydb/core/impl/BaseGrpcTransport.java

```diff
@@ -21,27 +21,36 @@
 import tech.ydb.core.grpc.GrpcStatuses;
 import tech.ydb.core.grpc.GrpcTransport;
 import tech.ydb.core.impl.auth.AuthCallOptions;
+import tech.ydb.core.impl.operation.OperationManager;
 import tech.ydb.core.impl.pool.GrpcChannel;
 import tech.ydb.core.impl.stream.EmptyStream;
 
 /**
- *
  * @author Aleksandr Gorshenin
  */
 public abstract class BaseGrpcTransport implements GrpcTransport {
     private static final Logger logger = LoggerFactory.getLogger(GrpcTransport.class);
 
-    private static final Result<?> SHUTDOWN_RESULT =  Result.fail(Status
+    private static final Result<?> SHUTDOWN_RESULT = Result.fail(Status
             .of(StatusCode.CLIENT_CANCELLED)
             .withIssues(Issue.of("Request was not sent: transport is shutting down", Issue.Severity.ERROR)
-    ));
+            ));
+
+    private final OperationManager operationManager = new OperationManager(this);
```

Comment:
In this case - instance of OperationManager will be created per every BaseGrpcTransport instance. It isn't good behavior, because usually we use multiply instances of BaseGrpcTrasnsport. Maybe it will be better to move the OperationManager construction to YdbGrpcTransport?

Reply:
BaseGrpcTrasnsport implements GrpcTransport, GrpcTransport has method OperationManager getOperationManager(). Maybe make static field with OperationManager?

Reply:
No, it is bad decision. Client application may use multiply instances of YdbTransport - and it is valid usage. In this case every YbdTransport must have own instance of OperationManager